.eslintrc

```diff
@@ -1,4 +1,5 @@
 {
+  // note you must disable the base rule as it can report incorrect errors
```

Comment:
These changes are no longer needed after #1211.

I tested the PR without changes to .eslintrc, and `yarn lint` reports no error.

Reply:
I mean you can simply not change the file at all.